Casks/menu-bar-splitter.rb

```diff
@@ -1,10 +1,10 @@
 cask "menu-bar-splitter" do
-  version "2.0"
-  sha256 "4f1db97404fe48eaef14b8fd05c938a6ca9e6c6cd546730e25e85b709ed8fa8a"
+  version "2.0.1"
+  sha256 :no_check
```

Comment:
```suggestion
  sha256 "3a48c97cb3594933c5b2f97b0e31373a45847b8aefec7aeceaf7f0ea350b789e"
```

@jwhamilton99 The file itself does not need to be versioned to allow using a checksum, only the URL. Because the URL does change for each version this will prevent there being shasum mismatches when the cask is updated.

Reply:
Ok, thank you. I'm still new to making and updating casks.